modules/post/multi/manage/shell_to_meterpreter.rb

```diff
@@ -168,15 +174,15 @@ def run
         psh_payload = 'Start-Sleep -s 1;' << psh_payload
 
         encoded_psh_payload = encode_script(psh_payload)
-        cmd_exec(run_hidden_psh(encoded_psh_payload, psh_arch, true))
+        cmd_exec(run_hidden_psh(encoded_psh_payload, psh_arch, true), nil, command_timeout, { 'Channelized' => false })
```

Comment:
Do we need channelized here? Channelized is only going to impact meterpreter sessions, but this is a powershell session 👀